app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,22 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def eat_favourite_dish(self) -> None:
+        print(f"{self.nickname} is eating {self._favourite_dish}")
```

Comment:
no need to write this method, because it is in the Dwarf

DRY(do not repeat yourself) principle should be followed